scripts/discover_devices.py

```diff
@@ -51,7 +51,8 @@
 Example output:
     Switcher devices broadcast a status message every approximately 4 seconds. This
     script listens for these messages and prints a serialized version of the to the
-    standard output, for example (note the ``device_id`` and ``mac_address`` properties)::
+    standard output, for example (note the device_id and mac_address properties):
```

Comment:
I think the discover_devices script is being updated in another PR and this modification might cause conflicts.
Perhaps it's better to skip the modifications in this file as they are purely semantic.
Unless I'm missing something.